.github/ISSUE_TEMPLATE/kubernetes_bump.md

```diff
@@ -34,21 +34,25 @@ changes should be cherry-picked to all release series that will support the new
     * Note: Only bump for Cluster API versions that will support the new Kubernetes release.
   * Prior art: #9160
```

Comment:
Thx for the updates here, look great! We also have to update some docs in release-tasks.md

I think it is mostly: Setup jobs and dashboards for a new release branch

Reply:
done :-)